client/app/routes/pas-form/edit.js

```diff
@@ -0,0 +1,14 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import { STATUSCODE, STATECODE } from '../../optionsets/package';
```

Comment:
curious if you can write `... from 'client/optionsets/package` here?

Reply:
i haven't figured out why but it's been back and forth for me... sometimes the compiler wants relative, sometimes absolute path.

Reply:
Hmm, I will give it a try and see what happens 😅😅 ...

Reply:
Both work, which is better, semantically?